From f90070281dddbe704540ed7f07711f37cff6815c Mon Sep 17 00:00:00 2001
From: Steve Best <sbest@redhat.com>
Date: Fri, 29 Aug 2014 18:14:36 -0400
Subject: [redhat] Kconfig: enable Random Number Generator hardware driver for PowerNV platform

Message-id: <20140829181436.27191.18328.sendpatchset@ibm-p7r2-01.lab.bos.redhat.com>
Patchwork-id: 89895
O-Subject: [PATCH RHEL7.1 BZ1123116 2/2] redhat Kconfig: enable Random Number Generator hardware driver for PowerNV platform
Bugzilla: 1123116
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

RHBZ#:
------
https://bugzilla.redhat.com/show_bug.cgi?id=1123116

Description:
------------
Enable Random Number Generator hardware driver for PowerNV platform

RHEL Version Found:
-------------------
7.0

Upstream Status:
----------------
n/a

----------------
Steve Best

Proposed Patch:
---------------

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/powerpc64/CONFIG_HW_RANDOM_POWERNV b/redhat/configs/generic/powerpc64/CONFIG_HW_RANDOM_POWERNV
new file mode 100644
index 0000000..7edb0ba
--- /dev/null
+++ b/redhat/configs/generic/powerpc64/CONFIG_HW_RANDOM_POWERNV
@@ -0,0 +1 @@
+CONFIG_HW_RANDOM_POWERNV=m
-- 
1.7.1